service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java

```diff
@@ -64,9 +68,23 @@ private void configureRouteClaimSubmitForFull() {
         .process(FunctionProcessor.fromFunction(saveToDbService::insertClaim))
         // Use Properties not Headers
         // https://examples.javacodegeeks.com/apache-camel-headers-vs-properties-example/
+        .setProperty("id", simple("${body.recordId}"))
         .setProperty("diagnosticCode", simple("${body.diagnosticCode}"))
+        .setProperty("veteranIcn", simple("${body.veteranIcn}"))
         .routingSlip(method(SlipClaimSubmitRouter.class, "routeClaimSubmit"))
-        .routingSlip(method(SlipClaimSubmitRouter.class, "routeClaimSubmitFull"));
+        .routingSlip(method(SlipClaimSubmitRouter.class, "routeClaimSubmitFull"))
+        .convertBodyTo(String.class)
+        .process(
+            new Processor() {
+              @Override
+              public void process(Exchange exchange) throws Exception {
+                String response = (String) exchange.getIn().getBody();
```

Comment:
I think you can use <T> T getIn(Class<T> type) with AssessmentResult here since you already defined that class (but also see the comment on that class0.  If you do that there is no need for veteranIcn and diagnosticCode below since AssessmentResult will have those.